samples/zigbee/ncp/README.rst

```diff
@@ -116,17 +116,33 @@ This can be configured by setting:
 * :kconfig:`CONFIG_ZBOSS_TRACE_USB_CDC_LOGGING` - to select nRF USB serial over UART serial.
 * :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME` - to select the serial device for printing Zigbee stack logs, set to ``"CDC_ACM_1"``.
 
-To create two instances of USB CDC ACM for application, set the following:
+To create two instances of USB CDC ACM for application, two entries in the dts overlay file for selected board are required, each for every USB CDC ACM instance.
+See :ref:`zephyr:usb_device_cdc_acm` for more information.
+NCP sample configures single USB CDC ACM instance ``"CDC_ACM_0"`` by default.
+To configure second USB CDC ACM instance ``"CDC_ACM_1"``, you can extend ``zephyr_udc0`` node in the dts overlay file:
 
-* :kconfig:`CONFIG_USB_CDC_ACM_DEVICE_COUNT` - defines number of USB CDC ACM devices to create, set to ``2`` in this configuration of NCP sample.
-* :kconfig:`CONFIG_USB_COMPOSITE_DEVICE` - to enable composite USB device driver.
+.. code-block:: console
+
+&zephyr_udc0 {
+   cdc_acm_uart0: cdc_acm_uart0 {
+      compatible = "zephyr,cdc-acm-uart";
+      label = "CDC_ACM_0";
+   };
+
+   cdc_acm_uart1: cdc_acm_uart1 {
+      compatible = "zephyr,cdc-acm-uart";
+      label = "CDC_ACM_1";
+   };
+};
+
+If you have configured two USB CDC ACM instances, enable composite USB device driver using :kconfig:`CONFIG_USB_COMPOSITE_DEVICE`.
```

Comment:
Does it mean that we have to update NCP sample configs?

Reply:
We don't. It's only used if second USB CDC ACM instance is configured to be used for logging binary ZBOSS traces over USB.